# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentParent.h
# Commit: 095b3edec3c8
# Full Hash: 095b3edec3c8ba9318c71d0acc91e10ce57818dc
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2019-04-29 21:53:38
# Regressor Bug: 1514869
# File Overlap Count: 1
# Description:
#   Bug 1514869 - patch 2 - Adapt platform-font-list code to work with either the existing in-process font list or cross-process shared font list. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D22938
# ==============================================================================

diff -r 21ef00977ab6 -r 095b3edec3c8 dom/ipc/ContentParent.h
--- a/dom/ipc/ContentParent.h	Sat Apr 27 15:37:29 2019 +0000
+++ b/dom/ipc/ContentParent.h	Sat Apr 27 15:37:58 2019 +0000
@@ -219,6 +219,8 @@
 
   static void BroadcastStringBundle(const StringBundleDescriptor&);
 
+  static void BroadcastFontListChanged();
+
   const nsAString& GetRemoteType() const;
 
   virtual void DoGetRemoteType(nsAString& aRemoteType,
@@ -277,6 +279,7 @@
   static void NotifyUpdatedDictionaries();
 
   static void NotifyUpdatedFonts();
+  static void NotifyRebuildFontList();
 
 #if defined(XP_WIN)
   /**